docs/quick-start/flashing-elrs.md

```diff
@@ -32,18 +77,93 @@ TODO
+
+Power your Flight Controller by either connecting a LiPo or attaching the USB cable (if receiver gets powered from USB via a 4v5 pad). Receiver's LED will blink slow at first, and after 20s or 30s (can be adjusted via ExpressLRS Configurator using `AUTO_WIFI_ON_INTERVAL`), it should blink fast indicating it's on Wifi Hotspot Mode.
+
+Connect to the Wifi Network the receiver has created. It should be named something like `ExpressLRS RX` with the same `expresslrs` password as the TX Module Hotspot. Navigate to the same web address as the TX Module (usually http://10.0.0.1). The Firmware upload page should load, and using the File Upload Form, navigate where the correct Receiver `firmware.bin` is (like with the Tx module, you can also drag-and-drop the firmware file into the form field or use the `Browse` or `Choose File` button). Click on **Update** button and the firmware file will be uploaded and the update process should commence. 
+
+A white page should load momentarily with the message **Update Success! Rebooting...**. Wait a little bit and the receiver should be updated. Power cycle and your module and receiver should now be bound (given you have updated the Tx Module as well, and that they have the same binding phrase and options).
+
+- Flashing via Passthrough
+
+Wire up your receiver to a free uart in your Flight Controller. The image above shows the pinouts for these receivers. Wire TX on receiver to an RX pad on the FC, and the RX on receiver to a TX pad on the FC in the same UART. Wire 5v and Gnd as normal (5v to a 5v pad on FC and Gnd to a Gnd pad on the FC).
+
+You will need to bridge the `Boot` pads on the receiver the first time you'll be updating via this method. The image above shows where the `Boot` pads are. A solid LED indicates the receiver is in `Bootloader` mode when the TX module is OFF (Solid LED also indicates Radio+module & Receiver is bound and has connection). 
+
+*Note: if you haven't bridged the `Boot` pads but the receiver has solid LED light, your FC is probably pulling the current UART's RX pad `LOW` which will interfere with the normal and passthrough flashing of this receiver. Find another UART and wire your receiver there instead*
+
+Bridging the `Boot` pads is no longer needed past 1.0.0-RC6. 
+
+Power your FC with a LiPo, or if receiver is powered via USB (receiver is connected to a 4v5 pad), connect the FC to your USB port. Using the ExpressLRS Configurator, with the correct Target selected and options set, click on **Build & Flash**. Wait for the process to finish and you should be greeted with the "Success" banner.
+
+Unplug USB and LiPo, and removed the solder on the bridged `Boot` pads. You no longer need it (past 1.0.0-RC6). Power your TX Module and then your FC to verify you are bound and has connection.
+
+#### Happymodel PP RXes

```

Comment:
I think only STM variant PP does not have wifi capabilities? They have ESP32 based ones and they support wifi just fine.

Reply:
Are there Happymodel PPs that were using ESP MCUs now?

Reply:
I thought PPs were exclusively STM32 yeah...

Reply:
Technically those are called EP1 / EP2.. 😄

Reply:
But this section is specific for the Happymodel PPs..